modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -111,28 +111,43 @@
 	<td></td>
 </tr>
 <tr>
```

Comment:
Could you check if URL_LABEL [in this line](https://github.com/liferay-headless/liferay-portal/blob/17f4abc2cdee9a7f2dc24648bc6527173c6e266e/modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path#L19) works here? And we can't use the http://localhost:8080 here since this could be changes